feature-group-definitions/svg2-script-html-equivalance.yml

```diff
@@ -0,0 +1,6 @@
+spec: https://svgwg.org/svg2-draft/interact.html#ScriptElement
+compat_features:
+  - svg.elements.script
```

Comment:
Probably minor but I would have thought the group is specifically about the 3 attributes only and not about the script element itself.

Reply:
```suggestion
```

No, that's a good catch. I'll remove this line.